test/sdk/coinstac/consortia.js

```diff
@@ -0,0 +1,95 @@
+/* jshint strict:false */
```

Comment:
I think we should keep with the convention of naming after req type here as well